recipes/_install-linux.rb

```diff
@@ -45,6 +45,21 @@
     action :upgrade
   end
 when 'rhel', 'fedora', 'amazon'
+  # Centos < 7 was deprecated on agent {6,7}.52
+  agent_major_version = Chef::Datadog.agent_major_version(node)
+  if agent_major_version.to_i >= 6 && platform_family?('rhel') && node['platform_version'].to_i < 7
```

Comment:
❓ Question :
in `platform_family` is there no `centos` and `rhel` only ? since I can see some `context 'on CentOS < 7 distro' do` expected to be one for centos 🤔

Reply:
https://docs.chef.io/infra_language/checking_platforms/#platform_family:
> redhat, centos, oracle, almalinux, rocky, scientific, xenserver, clearos, bigip, parallels, xcp, virtuozzo, alibabalinux, and ibm_powerkvm platforms

Reply:
I see indeed shouldn't be an issue 👍